### PR TITLE
Update bintray-sbt to 0.3.0

### PR DESCRIPTION
**Pull request by nightscape**
which is an auto-plugin and makes things a little easier

### CODE REVIEW

File: bintray.sbt [hunk 1]
```diff
@@ -1,6 +1,3 @@
-import bintray.Keys._
-
-bintrayPublishSettings
 publishMavenStyle := false
 bintrayOrganization in bintray := None
-repository in bintray := "sbt-plugins"
+bintrayOrganization := Some("sbt-plugins")
```
Comment (eed3si9n) [inline bintray.sbt]: Should be `bintrayRepository`

**Comment by nightscape** [comment]
Amended to the commit

**Comment by eed3si9n** [comment]
Travis says

```
/home/travis/build/sbt/sbt-assembly/bintray.sbt:3: error: type mismatch;
 found   : Some[String]
 required: String
bintrayRepository := Some("sbt-plugins")
                         ^
```

**Comment by nightscape** [comment]
Screw me! I amended again, now it's working.

**Pull request merged**